version.txt

```diff
@@ -1 +1 @@
-1.28.0
+1.29.0-rc.2
```

Comment:
You'll need to remove this from this commit. We need to get the dependencies updated on `master` then I can create the `release-1.29` branch that includes that commit -- *then* we'll cut a release on that branch.

Reply:
Missed the comment, got it sounds good to me